xscen/aggregate.py

```diff
@@ -169,22 +191,29 @@ def compute_deltas(
 
     # Separate the reference from the other horizons
     ref = ds.where(ds.horizon == reference_horizon, drop=True)
-    # Remove references to 'year' in REF
-    ind = pd.MultiIndex.from_arrays(
-        [ref.time.dt.month.values, ref.time.dt.day.values], names=["month", "day"]
-    )
-    ref = ref.assign(time=ind).unstack("time")
 
-    ind = pd.MultiIndex.from_arrays(
-        [
-            ds.time.dt.year.values,
-            ds.time.dt.month.values,
-            ds.time.dt.day.values,
-        ],
-        names=["year", "month", "day"],
-    )
-    other_hz = ds.assign(time=ind).unstack("time")
+    if "time" in ds:
+        # Remove references to 'year' in REF
+        ind = pd.MultiIndex.from_arrays(
+            [ref.time.dt.month.values, ref.time.dt.day.values], names=["month", "day"]
+        )
+        ref = ref.assign(time=ind).unstack("time")
+
+        ind = pd.MultiIndex.from_arrays(
+            [
+                ds.time.dt.year.values,
+                ds.time.dt.month.values,
+                ds.time.dt.day.values,
+            ],
+            names=["year", "month", "day"],
+        )
+        other_hz = ds.assign(time=ind).unstack("time")
 
+    else:
+        other_hz = ds.sel(
+            horizon=[h for h in ds.horizon.values if h != reference_horizon]
+        )
```

Comment:
I just realized this, but this removes the reference from the results, while the option with `time` doesn't (all `horizon` are still there at the just, just with 0/1 in the reference). Shouls we simply have this instead, to mimic the behaviour?
```suggestion
        other_hz = ds
```

Reply:
The point of keeping it when the time coordinate is present is to be able to plot a time series that doesn't have a hole at the reference horizon (eg. for climatedata.ca). I don't really see the point of it with warming levels because we won't do a timeserie and a map of 0/1 is not super interesting, but if you want it we can put it back.